tests/scripts/all.sh

```diff
@@ -199,6 +199,8 @@ pre_initialize_variables () {
     : ${ARMC6_BIN_DIR:=/usr/bin}
     : ${ARM_NONE_EABI_GCC_PREFIX:=arm-none-eabi-}
     : ${ARM_LINUX_GNUEABI_GCC_PREFIX:=arm-linux-gnueabi-}
+    : ${ARM_LINUX_GNUEABIHF_GCC_PREFIX:=arm-linux-gnueabihf-}
```

Comment:
The other similar options above each come with a command-line option (`--arm-none-eabi-gcc-prefix` and `--arm-linux-gnueabi-gcc-prefix`), should we have command-line options for the two options you're introducing as well?

(Not a blocker. I'm not even convinced we need the two command-line options that are already present - we're rarely going to need to override this, and when we do we can set the variables in the environment anyway. OTOH, consistency is usually a good thing.)